### PR TITLE
add copyright to source files

### PR DESCRIPTION
**Pull request by merobi-hub**
Signed-off-by: Michael Robinson <merobi@gmail.com>

### Problem

With few exceptions, the project lacked a copyright. Including one in source files is an expectation of the LFAI.

### Solution

This PR adds copyright lines to Python, Rust, Bash, and Java files throughout the project. 

- [ ] Your change modifies the [core](https://github.com/OpenLineage/OpenLineage/blob/main/spec/OpenLineage.json) OpenLineage model
- [ ] Your change modifies one or more OpenLineage [facets](https://github.com/OpenLineage/OpenLineage/tree/main/spec/facets)

If you're contributing a new integration, please specify the scope of the integration and how/where it has been tested (e.g., Apache Spark integration supports `S3` and `GCS` filesystem operations, tested with AWS EMR).

### Checklist

- [x] You've [signed-off](https://github.com/OpenLineage/OpenLineage/blob/main/why-the-dco.md) your work
- [ ] Your pull request title follows our [guidelines](https://github.com/OpenLineage/OpenLineage/blob/main/CONTRIBUTING.md#creating-pull-requests)
- [ ] Your changes are accompanied by tests (_if relevant_)
- [x] Your change contains a [small diff](https://kurtisnusbaum.medium.com/stacked-diffs-keeping-phabricator-diffs-small-d9964f4dcfa6) and is self-contained
- [ ] You've updated any relevant documentation (_if relevant_)
- [ ] You've updated the [`CHANGELOG.md`](https://github.com/OpenLineage/OpenLineage/blob/main/CHANGELOG.md) with details about your change under the "Unreleased" section (_if relevant, depending on the change, this may not be necessary_)
- [ ] You've versioned the core OpenLineage model or facets according to [SchemaVer](https://docs.snowplowanalytics.com/docs/pipeline-components-and-applications/iglu/common-architecture/schemaver) (_if relevant_)

### CODE REVIEW

**Comment by mobuchowski** [comment]
Can we split changes per file type, as in Marquez? https://github.com/MarquezProject/marquez/pull/1996/commits

**Review by mobuchowski — CHANGES_REQUESTED** [review]
In `spark` integration `bin` directory appeared, with compiled binary versions of java classes. This directory should be removed

**Comment by rossturk** [comment]
> In `spark` integration `bin` directory appeared, with compiled binary versions of java classes. This directory should be removed

@mobuchowski I think this has been addressed now, do you agree?

**Comment by merobi-hub** [comment]
@collado-mike, pinging you in case you have some time to review today. Thanks

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<!-- Copyright 2018-2022 contributors to the OpenLineage project -->
+
 <!-- SPDX-License-Identifier: Apache-2.0 -->
 
 <!-- SPDX-License-Identifier: CC-BY-4.0 -->
```
Comment (mobuchowski) [inline CONTRIBUTORS.md:5]: We get double licences here, with conflicting information

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+<!-- Copyright 2018-2022 contributors to the OpenLineage project -->
```
Comment (mobuchowski) [inline CONTRIBUTORS.md:1]: This information already is below:

`<!-- Copyright Contributors to the OpenLineage. -->`

File: integration/common/setup.py [hunk 3]
```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python
+# Copyright 2018-2022 contributors to the OpenLineage project
```
Comment (mobuchowski) [inline integration/common/setup.py:1]: Any comments need to be below shebangs

` # !/usr/bin/env python`

File: integration/dagster/setup.py [hunk 4]
```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python
+# Copyright 2018-2022 contributors to the OpenLineage project
```
Comment (mobuchowski) [inline integration/dagster/setup.py:1]: Same shebang issue here.

File: integration/dbt/setup.py [hunk 5]
```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python
+# Copyright 2018-2022 contributors to the OpenLineage project
```
Comment (mobuchowski) [inline integration/dbt/setup.py:1]: Same shebang issue here.

**Review by mobuchowski — CHANGES_REQUESTED** [review]

**Comment by mobuchowski** [comment]
I think this will work!

**Review by collado-mike — APPROVED** [review]
This all looks great to me!

**Comment by merobi-hub** [comment]
thanks, @collado-mike . Added the lines to some files I'd missed, but now I think it's ready to be merged

**Comment by merobi-hub** [comment]
All of a sudden the Spark integration tests started failing. Today I merged with main to see if that fixed the problem, but no.
Can you advise @collado-mike or @mobuchowski ? Thanks

**Comment by mobuchowski** [comment]
@merobi-hub you have few duplicated files, like `integration/spark/app/src/main/java/io/openlineage/spark/agent/EventEmitter 2.java`

**Review by mobuchowski — APPROVED** [review]

**Comment by mobuchowski** [comment]
@merobi-hub approved and merged

**Comment by merobi-hub** [comment]
Thank you, @mobuchowski !

**Pull request merged**